### DISCUSSION TITLE
Does promitor support access over multiple Azure ADs?

### DISCUSSION THREAD

**Discussion by adhiraj-g** [tomkerkhove/promitor#1642] [Q&A]
Hi,

As mentioned in the Docs, we need to create a Service Principal with appropriate roles in order to fetch out metrics.
How can we fetch details in case setups are over multiple Azure ADs? Is that support available on Promitor?

Do we need to create multiple Service Principals?

**Comment by tomkerkhove** [comment]
That is only supported by deploying multiple instances of Promitor that are configured to use different ADs with corresponding service principles.

Can you elaborate on your scenario?

**Reply by adhiraj-g** [reply]
Hi Tom,

The setup we have has separate AD setup for each client. Our objective is to monitor the azure resources associated to all the clients.

While searching for Service principal creation, multi-tenant applications are also available on Azure. Would that work in case of promitor?

Below, stackoverflow link, provides a descent overview regarding multi-tenant SPN.
https://stackoverflow.com/questions/66236243/can-we-access-different-azure-ad-directories-with-single-azure-ad-app
